client/src/components/Reviews/components/ReviewList.jsx

```diff
@@ -12,19 +12,39 @@ const List = styled.div`
   margin-top: 1em;
 `;
 
-const Buttons = styled.button`
+const AddReview = styled.button`
+  background: none!important;
+  border: none;
+  padding: 0!important;
   position: absolute;
-  right:  20%;
+  right:  40%;
+  bottom:  2%;
+`;
+const MoreReviews = styled.button`
+  background: none!important;
+  border: none;
+  padding: 0!important;
+  position: absolute;
+  right:  60%;
   bottom:  2%;
 `;
 
 function ReviewList() {
+  function AddReview(e) {
+    e.preventDefault();
+    // TODO this will pull up the review modal window to allow for the user to add a review
+  }
+  function MoreReviews(e) {
+    e.preventDefault();
+    // TODO make a request to the api to get more/will change which reviews are being displayed
+  }
   return (
     <List>
       <div>there are some reviews that are not sorted</div>
       <Review />
       <Review />
-      <Buttons>Test</Buttons>
+      <AddReview onClick={AddReview}> Add review</AddReview>
```

Comment:
You're using AddReview to refer to two different things here. Doesn't this cause an error? I would suggest being more explicit with your naming, e.g. AddReviewButton and handleAddReviewClick. I try to prefix my event handlers with "handle..." in an attempt to be more explicit about what the purpose of the function is.

Reply:
ah I didn't even notice that. yeah ill change that before I merge anything